tests/data/ar/Animal.php

```diff
@@ -38,10 +38,10 @@ public function getDoes()
 
     /**
      *
-     * @param type $row
+     * @param array $row
      * @return \yiiunit\data\ar\Animal
      */
-    public static function instantiate($row)
+    public static function instantiate($row = [])
```

Comment:
As an alternative without bc breaking we can leave `instantiate($row)` method as is and make another method like `instantiateInternal()` without any parameters.